pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling"]
+requires = ["hatchling", "numpy>=2.0"]
```

Comment:
Are we requiring numpy 2 now?

Reply:
Only for building the distribution..  2.x is not required for installing in your local environment.

You can still pip install openmdao into an environment with NumPy 1.x, as seen in the CI workflow